app/routes/api.circulars.ts

```diff
@@ -166,8 +184,15 @@ export async function action({ request }: ActionFunctionArgs) {
   } = await parseAccessToken(bearer)
 
   // Make sure that the access token contains the required scope for this API
-  if (!scope.split(' ').includes(submitterGroup))
-    throw new Response('Invalid scope', { status: 403 })
+  switch (request.method) {
+    case 'POST':
+      if (!scope.split(' ').includes(submitterGroup))
+        throw new Response('Invalid scope', { status: 403 })
+
+    case 'PUT':
+      if (!scope.split(' ').includes(moderatorGroup))
+        throw new Response('Invalid scope', { status: 403 })
+  }
```

Comment:
Add a default clause.

Reply:
so we have two options here. Currently it does have a check (it's not a default case, but a check that the method is either put or post) on line 174 https://github.com/nasa-gcn/gcn.nasa.gov/pull/3105/files#diff-62f353c505e0f6ee6f4790fa20b2ccd3d9dce7ae11b05797df5b061ddf8a2d47R174
so a default case would never be hit.

if you want a default method, that's fine, i can remove line 174 and then put in a default case. I would have to break up the rest of the code because it would become unreachable if that section has a default case, so I would just have to rearrange things so that each case block calls the function to do the updating. 

would you like to keep line 174 checking for that, or do you want me to rearrange the code and add a default case?